packages/react-data-grid/src/Cell.js

```diff
@@ -272,14 +303,38 @@ class Cell extends React.Component {
     return null;
   }
 
+  didDependentValuesChange = (nextProps) => {
+    let currentColumn = this.props.column;
```

Comment:
Am wondering if we can do away with the whole shouldComponentUpdate function and just make `Cell` a PureComponent. It looks like we are just comparing each prop anyway, and it would be much simpler to maintain